libs/backend/src/logs/logs_api.ts

```diff
@@ -0,0 +1,48 @@
+import { LogsApi } from '@votingworks/types';
+import { UsbDrive } from '@votingworks/usb-drive';
+import * as fs from 'fs/promises';
+import { join } from 'path';
+
+const LOG_DIR = '/var/log/votingworks';
+
+/** Creates a shareable implementation of {@link LogsApi}. */
+export function createLogsApi({
+  usbDrive,
+  machineId,
+}: {
+  usbDrive: UsbDrive;
+  machineId: string;
+}): LogsApi {
+  return {
+    async exportLogsToUsb(): Promise<boolean> {
```

Comment:
Instead of returning a boolean to indicate success vs. failure, this might be a good use case for the [`Result` class](https://github.com/votingworks/vxsuite/blob/main/libs/basics/src/result.ts), e.g. `Result<void, 'no-log-directory' | 'no-usb-drive'>`.

On error, we'd return `err('no-log-directory')`, `err('no-usb-drive')`, etc. And on success, we'd return `ok()`.

Consumers could then condition on `result.isOk()`.

Reply:
yep, I definitely had on my list a "need to fix this" but wasn't sure yet how to do it, so your pointer is perfect.